dev/com.ibm.ws.container.service/bnd.bnd

```diff
@@ -22,13 +22,17 @@ Include-Resource: \
 # file, and place an @version javadoc tag in package-level javadoc. 
 # Append ";provide:=true" if this bundle also provides an implementation
 # for the exported API.
+
 Export-Package: \
    com.ibm.ejs.j2c, \
    com.ibm.ejs.util, \
    com.ibm.ejs.util.dopriv, \
    com.ibm.jtc.adapter;provide:=true, \
    com.ibm.websphere.csi;-split-package:=merge-last, \
    com.ibm.ws.container.service.annotations; provide:=true, \
+   com.ibm.ws.container.service.annotations.internal; provide:=true, \
+   com.ibm.ws.container.service.annocache; provide:=true, \
+   com.ibm.ws.container.service.annocache.internal; provide:=true, \
```

Comment:
Why are we exporting packages with the word internal in them?  Should these truly be exported?

Reply:
These are currently necessary because not all consumers of the annotation container services work through the top level types.  This requires relocation of consumer code into the container service component to fix.

Reply:
See: APSFOUND-1 Followup: Refactor annotation container service to consolidate exposed APIs #9215